flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/RescaleParallelismQueryParameter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.messages;
+
+/**
+ * Parallelism for the rescaling of jobs specified as a {@link MessageQueryParameter}.
+ */
+public class RescaleParallelismQueryParameter extends MessageQueryParameter<Integer> {
+
+	public static final String KEY = "parallelism";
+
+	protected RescaleParallelismQueryParameter() {
+		super(KEY, MessageParameterRequisiteness.MANDATORY);
+	}
+
+	@Override
+	public Integer convertValueFromString(String value) {
+		return Integer.valueOf(value);
```

Comment:
We could already validate that the integer is greater than 0. 
Also there could be validation in `Dispatcher#rescaleJob` and `JobMaster#rescaleJob`

Reply:
Class was renamed to `RescalingParallelismQueryParameter`

Reply:
Will add it to the `JobMaster`.

Reply:
ok